openshift.yaml

```diff
@@ -15,6 +15,14 @@ parameters:
     description: >
       The version of OpenShift Container Platform to deploy
 
+  # What version of OpenStack Platform to install
+  # This value is used to select the RPM repo for the OSP release to install
+  osp_version:
+    type: string
+    description: >
+      The version of OpenStack Platform to use to collect data
+
+
```

Comment:
Could you please set a default value here (and remove the extra newline)? Otherwise it looks like ready to be merged.